biber/src/main/java/com/petros/bibernate/entity/Buyer.java

```diff
@@ -0,0 +1,28 @@
+package com.petros.bibernate.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.petros.bibernate.annotation.*;
```

Comment:
check import options in your IDE like here:
![image](https://github.com/petros-educational-society/bibernate/assets/17574469/cfced263-d404-415b-8126-ef4612384820)
your IDE groups import into "*" automatically, because of setup options. 
fix options in IDE anf split imports with "*" in the code

Reply:
This setting will affect my work project((  I think we can leave as it is.